centreon/src/Core/Security/Authentication/Infrastructure/Provider/SAML.php

```diff
@@ -156,13 +156,22 @@ public function authenticateOrFail(LoginRequest $request): void
             throw InvalidUserIdAttributeException::create();
         }
 
+        $samlNameIdFormat = $auth->getNameIdFormat();
+        $samlNameIdNameQualifier = $auth->getNameIdNameQualifier();
+        $samlNameIdSPNameQualifier = $auth->getNameIdSPNameQualifier();
+
         $this->username = $attrs[0];
+
         CentreonSession::writeSessionClose('saml', [
             'samlSessionIndex' => $auth->getSessionIndex(),
             'samlNameId' => $auth->getNameId(),
+            'samlNameIdFormat' => ! empty($samlNameIdFormat) ? $samlNameIdFormat : null,
+            'samlNameIdNameQualifier' => ! empty($samlNameIdNameQualifier) ? $samlNameIdNameQualifier : null,
+            'samlNameIdSPNameQualifier' => ! empty($samlNameIdSPNameQualifier) ? $samlNameIdSPNameQualifier : null,
```

Comment:
Nitpick coding style

```suggestion
            'samlNameIdFormat' => $auth->getNameIdFormat() ?: null,
            'samlNameIdNameQualifier' => $auth->getNameIdNameQualifier() ?: null,
            'samlNameIdSPNameQualifier' => $auth->getNameIdSPNameQualifier() ?: null,
```